harvard-library/librarycloud_ingest
# Ltsiiif 1301

**XSL upgrades for the BTA project**
* * *

**JIRA Ticket**: [(link)](https://jira.huit.harvard.edu/browse/LTSIIIF-1301)

# What does this Pull Request do?
This pr adds xsl enhancements required for the Black Teacher Archive project.

# How should this be tested?

The project should build with a mvn clean install; openjdk 8 required
Deploy to qa and ask for review. Done by Michael and Robin

# Test coverage
Yes/No: Are changes in this pull-request covered by:
- unit tests? Y
- integration tests?
